app/src/lib/feature-flag.ts

```diff
@@ -132,3 +132,8 @@ export function enableLineChangesInCommit(): boolean {
 export function enableWindowsOpenSSH(): boolean {
   return enableBetaFeatures()
 }
+
+/** Should we use SSH askpass? */
+export function enableSSHAskPass(): boolean {
+  return __WIN32__ && enableBetaFeatures()
```

Comment:
Nit: Should the windows check be in the feature-flag? Just figured that once this goes in and someday we remove the feature flag.. will have to move the Windows check to inside the code somewhere.

Reply:
Yeah, I'm expecting to have something for macOS soon and remove that Win32 check, so it should be ok 🤞